qiskit_bot/api.py

```diff
@@ -149,7 +149,16 @@ def on_pull_event(data):
     if data['action'] == 'opened':
         repo_name = data['repository']['full_name']
         pr_number = data['pull_request']['number']
+        pr = REPOS[repo_name].gh_repo.get_pull(pr_number)
         if repo_name in REPOS:
+            if data['pull_request']['author_association'] != 'MEMBER':
+                # tag PR with 'community PR' label & notify reviewers
+                labels = pr.get_labels()
+                label_names = [label.name for label in labels]
+                if "Community PR" not in label_names:
+                    pr.add_to_labels("Community PR")
+                    pr.create_review_request(
+                        team_reviewers="community-reviewers")
```

Comment:
The docs for this say `team_reviewers` takes a list of strings: https://pygithub.readthedocs.io/en/latest/github_objects/PullRequest.html#github.PullRequest.PullRequest.create_review_request did you test that this call works with a single string?

Reply:
hey I haven't tested this call explicitly, I'm not exactly sure what's the best way to test these pygithub functions (I'm much more used to using the regular github js api 😅 ). Do you have a suggestion for how to best validate these work as expected? I'm in the process of adding some unit tests, but feels a bit flimsy just mocking everything :/